screen-orientation/lock-basic.html

```diff
@@ -15,13 +15,17 @@
 
 promise_test(async t => {
   await test_driver.bless("request full screen", () => {
-      return document.documentElement.requestFullscreen();
-    });
+    return document.documentElement.requestFullscreen();
+  });
   const value = await screen.orientation.lock('any');
   assert_equals(value, undefined);
+  document.exitFullscreen();
```

Comment:
I think we want `return document.exitFullscreen();` here, as exitFullscreen() returns a promise. 

Applies to others too!

Reply:
Sorry! 😊